server/routes/getRewards.ts

```diff
@@ -18,26 +21,20 @@ interface IEventData {
 }
 
 const getRewards = async (models, req: Request, res: Response, next: NextFunction) => {
-  const { chain, stash } = req.query;
+  const { chain, stash_id } = req.query;
   let { startDate, endDate } = req.query;
+  let { version } = req.query;
+  
+  // variables
+  let validators = {};
+  let rewards;
+  let session_rewards;
 
-  if (!chain) return next(new Error(Errors.ChainIdNotFound));
-
-  let eraLengthInHours;
-  if (chain === 'edgeware') {
-    eraLengthInHours = 6;
-  } else if (chain === 'kusama') {
-    eraLengthInHours = 6;
-  }
+  if (!chain) { return next(new Error(Errors.ChainIdNotFound)); }
 
-  const chainInfo = await models.Chain.findOne({
-    where: { id: chain }
-  });
-  if (!chainInfo) {
-    return next(new Error(Errors.InvalidChain));
-  }
+   //set default values 
+  if (!version) { version = 33; }
```

Comment:
these are chain specific btw, this should only be the case for Edgeware, which is currently at version 31 and will be at version 38 soon.

Reply:
fixed in latest commit, setting default to 31 for edgeware.